FrontEnd/src/components/SignUp/components/signup/signup-form/SignUpFormContent.module.css

```diff
@@ -95,6 +97,35 @@
   background: var(--main-white, #fff);
 }
 
+.signup-form__field__password {
+  display: flex;
+  height: 22px;
+  padding: 5px 12px;
+  align-items: center;
+  gap: 4px;
+  align-self: stretch;
+  border-radius: 2px;
+  border: 1px solid #d9d9d9;
+  background: #fff;
```

Comment:
Do we have any shared variables for colors?

Reply:
Fixed.
But could you explain please what is the difference between hex color in styles and variables for colors?